dev/.rat-excludes

```diff
@@ -138,3 +138,5 @@ over10k
 exported_table/*
 ansible-for-test-node/*
 node_modules
+application_1656321732247_0006_1
```

Comment:
Have checked existing event logs in `spark/core/src/test/resources/spark-events`. It seems that each of them is added separately in `.rat-excludes`. So I personally think it may be better to follow the current pattern.

<img width="193" alt="image" src="https://user-images.githubusercontent.com/10705175/176214688-ce564382-4415-47a2-8b3d-0d587797296f.png">

Reply:
This still needs to be addressed